ui/src/main/webapp/src/app/core/authentication/windup.http.service.ts

```diff
@@ -64,6 +72,25 @@ export class WindupHttpService extends Http {
                 );
             });
         });
+
+        let responseObservable2 = responseObservable.do(
+            null,
+            () => {
+                console.warn("HTTP request FAILED");
+                this._eventBus.fireEvent(new LoadingSomethingFailedEvent(responseObservable))
+            },
+            () => {
+                if (this._eventBus) {
```

Comment:
these `if (this._eventBus) ` statements are not necessary and should not be there.

I guess you forgot to remove them.